grails-app/services/au/org/ala/ecodata/ElasticSearchService.groovy

```diff
@@ -1166,6 +1166,18 @@ class ElasticSearchService {
             // todo: Check if BioCollect requires all sites in `sites` property. If no, merge `projectArea` with `sites`.
             projectMap.projectArea = siteService.getSimpleProjectArea(projectMap.projectSiteId)
             projectMap.containsActivity = activityService.searchAndListActivityDomainObjects([projectId: projectMap.projectId], null, null, null, [max: 1, offset: 0])?.totalCount > 0
+
+            List projectActivities = projectActivityService.search([projectId: projectMap?.projectId])
+            if (projectActivities?.size() > 0) {
```

Comment:
A slight change in logic. `publicParticipation` should be true if any survey is publicAccess `true` and published is `true`.